src/Microsoft.SqlTools.ServiceLayer/DacFx/Contracts/DacFxDeployRequest.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+using Microsoft.SqlTools.ServiceLayer.TaskServices;
+using Microsoft.SqlTools.ServiceLayer.Utility;
+
+namespace Microsoft.SqlTools.ServiceLayer.DacFx.Contracts
```

Comment:
Do you think it makes sense to have so many references in file, folder, namespace, and class names to `DacFx`?  Instead of having more generic names like `DeploymentService` or `ExportService` etc.  For example, we don't call out SMO for all the classes that depend on that library.

Reply:
My opinion - I like have a DacFxService, which will wrap all of DacFx operations: import, export, schema compare, extract, deploy, etc.. However I don't think we need a DacFx prefix to all request/response type names in the RPC layer.

Reply:
I removed the DacFx prefix from everything except DacFxService and the DacFxOperation base class that is used by import, export, extract, and deploy operations